src/rgw/rgw_rest_s3.h

```diff
@@ -264,6 +264,7 @@ class RGWPostObj_ObjStore_S3 : public RGWPostObj_ObjStore {
   parts_collection_t parts;
   std::string filename;
   std::string content_type;
+  std::string storage_class;
```

Comment:
since we seem to be copying the storage class instead of referencing it in the body, is there a necessity to store storage_class as a member?